htlcswitch/switch.go

```diff
@@ -101,6 +100,18 @@ const (
 	CloseBreach
 )
 
+// PendingUpdate describes the pending state of a closing channel.
+type PendingUpdate struct {
+	Txid        []byte
```

Comment:
Missing `godoc` field comments here and below.

Reply:
Moved structs and made internal.